src/views/DrawerNavigatorItems.tsx

```diff
@@ -54,13 +67,24 @@ const DrawerNavigatorItems = ({
       const scene = { route, index, focused, tintColor: color };
       const icon = renderIcon(scene);
       const label = getLabel(scene);
-      const accessibilityLabel = typeof label === 'string' ? label : undefined;
+      const accessibleLabel =
+        typeof accessibilityLabel === 'string'
+          ? accessibilityLabel
+          : typeof label === 'string'
+          ? label
+          : undefined;
+      const accessibleHint =
+        typeof accessibilityHint === 'string' ? accessibilityHint : undefined;
+      const accessibleRole =
```

Comment:
What do you need to set the accessibility role to? I believe setting it to `"button"` by default should be enough. Do you need it to be customizable?

Reply:
I think button might be appropriate, but I could see others wanting link.

Reply:
We can add when people want it